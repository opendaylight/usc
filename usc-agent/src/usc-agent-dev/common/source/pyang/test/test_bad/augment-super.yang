 module augment-super {
	namespace "http://example.com/augmentsuper";
	prefix "as";

	include augment;

	augment "/as:interfaces" {
		leaf ifEntry {
			type int8;
		}
	}

	augment "/as:interfaces" {
          container x {
            container y {
              leaf foo {
                type int8;
                mandatory true; // error, mandatory node
              }
            }
            list z {
              key k;
              min-elements 2; // error, mandatory node
              leaf k {
                type int32;
              }
              leaf t {
                type int32;
                mandatory true;
              }
            }
            grouping bar {
              leaf q {
                type int32;
                mandatory true;
              }
            }
            uses bar; // error, mandatory node
          }
	}
}



