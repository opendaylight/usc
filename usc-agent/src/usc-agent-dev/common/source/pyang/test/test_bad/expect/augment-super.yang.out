augment-super.yang:8: error: DUPLICATE_CHILD_NAME
